<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BurnIsoDialog</class>
 <widget class="QDialog" name="BurnIsoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Burn ISO File</string>
  </property>
  <widget class="QLabel" name="IsoFileLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>ISO File</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="IsoPath">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="SelectIso">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </widget>
  <widget class="QLabel" name="WritersLabel">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>70</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Drive:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Writers">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>331</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="BurnISO">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>471</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Burn</string>
   </property>
  </widget>
  <widget class="QLabel" name="StatusLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Status:</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="StatusBurn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>270</y>
     <width>401</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>471</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Session options</string>
   </property>
   <widget class="QCheckBox" name="MultiSession">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Multi Session Disc</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="SingleSession">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Single Session Disc</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>481</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="RecSpeedLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Record Speed</string>
   </property>
  </widget>
  <widget class="QComboBox" name="RecordSpeed">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>291</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
